SafeExamBrowser.I18n/Data/de.xml

```diff
@@ -31,7 +31,7 @@
         Drucken ist nicht erlaubt gemäss der aktiven Konfiguration.
     </Entry>
     <Entry key="Browser_Tooltip">
-        Browser Applikation
+        Geöffnete Websites
```

Comment:
Ah, I see. I guess we leave it that way for now, though it may be a good idea to make the text dynamic in the future (depending on whether the SEB browser has open windows or not) @danschlet?